### DISCUSSION TITLE
Fragility of Solid

### DISCUSSION THREAD

**Discussion by pierre-H** [solidjs/solid#521] [General]
Hello,

First, thank you @ryansolid for your great work !

I am very interested in Solid, but I notice that you are the only main maintainer of the project.
This makes the project very fragile and, in my opinion, difficult to adopt in companies.
I understand 100% that it's necessary to start with one developer 😄 , but what about the future ?
What's your opinion about that ?

**Comment by ryansolid** [comment]
If you look at the organization and move out of this individual repo you will see more activity. We've been expanding the core team. There are 4 of us now. For instance, I did not create the Website, the REPL, the Bundler Plugins, the Starter templates.

My main contribution is just on this single repo where yes I do not believe others have been as comfortable contributing to both given the complexity and the focus on performance. But I imagine this won't always be the case. If you look at Vue or Svelte which are single-maintainer led they've managed to get past that and with adoption so will we.

**Reply by pierre-H** [reply]
Thank you for your answer and you're great work !
